test/run-openshift-pytest

```diff
@@ -8,4 +8,4 @@
 
 THISDIR=$(dirname ${BASH_SOURCE[0]})
 
-cd "${THISDIR}" && python3 -m pytest --showlocals -vv test_*.py
+cd "${THISDIR}" && python3 -m pytest -s -rxfapP --showlocals -vv test_*.py
```

Comment:
Could you please explain this part: `-rxfapP`

I see this in the help text of pytest:

```
  -r chars              Show extra test summary info as specified by chars: (f)ailed,
                        (E)rror, (s)kipped, (x)failed, (X)passed, (p)assed, (P)assed with
                        output, (a)ll except passed (p/P), or (A)ll. (w)arnings are enabled
                        by default (see --disable-warnings), 'N' can be used to reset the
                        list. (default: 'fE').
```

If I understand it correctly, the `a` means `all except passed` and then you have `pP` to include passed which basically means everything. Can this be replaced by `-rA`?.

Reply:
For some kind of reason, it does not work. But now, it is working. https://github.com/sclorg/s2i-python-container/pull/694/commits/5d908f097e966eac3f8053e137b2fd5a67547976

Reply:
@frenzymadness Well, It looks like extra summary info is not shown. See test results here: https://artifacts.osci.redhat.com/testing-farm/b786a35c-ed0a-457e-a35a-6e622cf2a4da/

Reply:
Locally I can see this:
```
=============================================================================================== short test summary info ===============================================================================================
PASSED test_deploy_templates.py::TestDeployTemplate::test_python_template_inside_cluster[django-postgresql.json]
PASSED test_deploy_templates.py::TestDeployTemplate::test_python_template_inside_cluster[django-postgresql-persistent.json]
PASSED test_imagestreams_quickstart.py::TestImagestreamsQuickstart::test_python_template_inside_cluster[django-postgresql.json]
PASSED test_imagestreams_quickstart.py::TestImagestreamsQuickstart::test_python_template_inside_cluster[django-postgresql-persistent.json]
PASSED test_python_ex_standalone.py::TestPythonExTemplate::test_python_ex_template_inside_cluster
PASSED test_python_ex_template.py::TestPythonExTemplate::test_python_ex_template_inside_cluster
```